### ISSUE TITLE
Some roundstart computers don't have functionality or circuit boards, allowing you to spam screw/unscrew them. 

### ISSUE THREAD

**Issue by TimrodDX** [space-wizards/space-station-14#3584]
How to reproduce:

Step 1. Locate a computer and attempt to use a screwdriver on it. This also works with the power drill in screw mode.
Step 2. The screen will be unscrewed (expected behavior) but the computer is now permanently disabled. Using a screwdriver or the power drill on it again will play the sound effect, but it won't actually do anything.
Step 3. Pull computer to bar (assuming it doesn't flip out and pin you to a wall) and then annoy people by spamming the power drill noise.

**Comment by Zumorica** [comment]
Can't replicate, but it seems the computer was missing the boad. 
What computer did you do this on, was it on roundstart or spawned with the entity spawner?

**Comment by Zumorica** [comment]
Nevermind, I have replicated this with the empty computers that have no board or functionality.

**Comment by TimrodDX** [comment]
I did it on the communications console in the captain's office first on US West during a live round, and then switched to EU West 2 and attempted it with the medical records computer in Medbay because it was the first one I came across.

**Comment by TimrodDX** [comment]
Yeah, I just tested it again and can confirm this also works on working computers. I did it at round start in sandbox mode with the ID computer in the HoP's office and again with the communications computer in the captain's office. 

I should note that doing this on certain computers - I've tested this with the cargo console on the bridge and the computers in the captain's office - can cause you to desync until you reconnect to the server. The ID computer in the HoP's office is consistent.
